lib/shared-sku-app/src/SkuPicker/SkuPicker.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from 'react';
+import get from 'lodash/get';
+import clamp from 'lodash/clamp';
+import debounce from 'lodash/debounce';
+import { Button, TextInput, Icon } from '@contentful/forma-36-react-components';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+import { Divider } from '../Divider';
+import { ProductList } from './ProductList';
+import { Paginator } from './Paginator';
+import { Product } from '../interfaces';
+import { Pagination } from './interfaces';
+import { ProductSelectionList } from './ProductSelectionList';
+import { styles } from './styles';
+
+export interface Props {
+  sdk: AppExtensionSDK;
+  fetchProductPreviews: Function;
+  fetchProducts: Function;
+}
+
+interface State {
+  activePage: number;
+  search: string;
+  pagination: Pagination;
+  products: Product[];
+  selectedProducts: Product[];
+  selectedSKUs: string[];
+}
+
+const SEARCH_DELAY = 250;
+
+function getSaveBtnText(selectedSKUs: string | string[]): string {
+  if (typeof selectedSKUs === 'string') {
+    return 'Save product';
+  }
+
+  switch (selectedSKUs.length) {
+    case 0:
+      return 'Save products';
+    case 1:
+      return 'Save 1 product';
```

Comment:
Doesn't anymore.. initially the idea was that if you're operating on a single Short text field you can't select more than 1 product so there's no point in saying "Save 1 product" if it can never be more than 1